app/views/users/_avatar_list.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="row">
+  <div class="mu-avatar-list">
+    <% avatars_for(current_user).sort_by(&:id).each do |it| %>
+      <img src="<%= it.image_url %>" alt="<%= it.description %>" mu-avatar-id="<%= it.id %>" class='mu-avatar-item'>
```

Comment:
We could extract this one to a helper

Reply:
Good, I'm making use of the already existing `AvatarHelper`